src/Console/ConfigurationResolver.php

```diff
@@ -480,6 +519,15 @@ function (FixerInterface $fixer) {
         }
     }
 
+    private function getFixersConflictMap()
+    {
+        return array(
```

Comment:
```
    const CONFLICTS = [
        'align_double_arrow'              => 'unalign_double_arrow',
        'align_equals'                    => 'unalign_equals',
        'concat_with_spaces'              => 'concat_without_spaces',
        'echo_to_print'                   => 'print_to_echo',
        'long_array_syntax'               => 'short_array_syntax',
        'no_blank_lines_before_namespace' => 'single_blank_line_before_namespace',
        'no_duplicate_semicolons'         => 'no_empty_statement',
        'phpdoc_type_to_var'              => 'phpdoc_var_to_type',
        'psr0'                            => 'psr4',
    ];
```

that's the full list of conflicts i have, excluding the two general fixers we have